doc/user-manual/language/instance-arguments.lagda.rst

```diff
@@ -418,6 +418,17 @@ flag.  Be aware that enabling this flag might lead to an exponential
 slowdown in instance resolution and possibly (apparent) looping
 behaviour.
 
+.. _qualified-instances:
+
+Qualified instances
++++++++++++++++++++
+
+By default, Agda considers all imported instances as candidates. For
+example, ``import Data.Nat.Instances`` will make Agda consider all
+instances from the module ``Data.Nat.Instances`` even if the module is
+not opened. This behaviour can be changed via the
+``--no-qualified-instances`` flag.
```

Comment:
Can you add something about instances in local modules?

Reply:
Yes, done.